### ISSUE TITLE
[SUGGESTION] | Priestess of Elune Marriage

### ISSUE THREAD

**Issue by Hamstarox** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1021]
**Describe your suggestion in full detail below:**
Priestesses of Elune should be allowed to marry (There is no known reason for them to be disallowed to marry. Tyrande is allowed to marry Malfurion but in lore they do not officially marry until Cataclysm in Darnassus after Malfurion has awoken.)

**Comment by zumbak04** [comment]
> Priestesses of Elune should be allowed to marry (There is no known reason for them to be disallowed to marry. Tyrande is allowed to marry Malfurion but in lore they do not officially marry until Cataclysm in Darnassus after Malfurion has awoken.)

If by Priestesses of Elune you mean only Tyrande and not all night elven theocracies, it's possible, but Tyrande needs her own feudal government type.

If you mean all night elven theocracies, it's not possible.
There is interesting bug in vanilla: monk, horse, eunuch barons can marry normally though these traits forbid marriages.
The only way to stop it is to give baron theocratic government with banned priest marriages. So, all religions in the mod must ban priest marriages.

**Comment by Hamstarox** [comment]
Im not sure that worrying about monk/eunuch/horse (is horse even in the GoA mod as a culture that occurs naturally?) is a big deal. Does it also extend to other traits that prevent marriage?

Also it was about all priest characters in general (unless a religion specifically banned it)

**Comment by zumbak04** [comment]
> Does it also extend to other traits that prevent marriage?

Yep, barons try to make impossible in the mod marriage... like n'raqi x human. Event breaks this marriage, gives n'raqi trait that forbids marriages. N'raqi baron ignores it and does marriage again.

Plus, unlike other rulers, barons don't find spouse, but spawn them. So, barons with such traits turn into character generators that hurt the game performance.
